test/TestCaseReader.cpp

```diff
@@ -23,12 +23,15 @@
 #include <boost/algorithm/string.hpp>
 #include <boost/range/adaptor/map.hpp>
 #include <boost/throw_exception.hpp>
+#include <boost/filesystem.hpp>
 
 using namespace std;
 using namespace solidity::frontend::test;
 
+namespace fs = boost::filesystem;
+
 TestCaseReader::TestCaseReader(string const& _filename):
-	m_file(_filename)
+	m_file(_filename), m_filename(_filename)
```

Comment:
What is the difference between `m_file` and `m_filename`?

Reply:
Ah, `m_file` is `std::ifstream`, just hard to tell that here in the constructor.

Reply:
Ok, I think I will rename `m_file` to `m_filestream`.